tests/utils.py

```diff
@@ -0,0 +1,30 @@
+from contextlib import contextmanager
+import logging
+
+
+def retry(
+    operation,
+    times=1,
+    wait=1,
+    error_msg=None,
+    logger='tests.utils.retry',
+):
+    log = logging.getLogger(logger)
```

Comment:
Create this as a singleton at module scope.

Reply:
The point was that callers of this `retry` function could provide their own logger name (could be useful if we start having retries within other retries).

Reply:
But I'll use a singleton as you requested and pass some string to put in logs instead.